sonoff/sonoff.h

```diff
@@ -160,13 +160,13 @@ typedef unsigned long power_t;              // Power (Relay) type
 #endif
 
 //enum ws2812NeopixelbusFeature { NEO_RGB, NEO_GRB, NEO_BRG, NEO_RBG, NEO_3LED, NEO_RGBW, NEO_GRBW };  // Doesn't work
-#define NEO_RGB                0            // Neopixel RGB leds
-#define NEO_GRB                1            // Neopixel GRB leds
-#define NEO_BRG                2            // Neopixel BRG leds
-#define NEO_RBG                3            // Neopixel RBG leds
-#define NEO_3LED               4            // Placeholder to test for 4 led types
-#define NEO_RGBW               5            // Neopixel RGBW leds
-#define NEO_GRBW               6            // Neopixel GRBW leds
+#define WS2812_NEO_RGB         0            // Neopixel RGB leds
+#define WS2812_NEO_GRB         1            // Neopixel GRB leds
+#define WS2812_NEO_BRG         2            // Neopixel BRG leds
+#define WS2812_NEO_RBG         3            // Neopixel RBG leds
+#define WS2812_NEO_3LED        4            // Placeholder to test for 4 led types
+#define WS2812_NEO_RGBW        5            // Neopixel RGBW leds
+#define WS2812_NEO_GRBW        6            // Neopixel GRBW leds
 
```

Comment:
I renamed the enum because it clashes with an enum in the Adafruit_Seesaw library that was added.